setup.py

```diff
@@ -5,19 +5,25 @@
 
 setuptools.setup(
     name="pyminio",
-    version="0.1.0",
+    version="0.2.0",
     author="Michael Tugendhaft",
     author_email="tugmica@gmail.com",
     description="Python client for Minio",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/mmm1513/pyminio",
     packages=setuptools.find_packages(),
-    install_requires=[],  # TODO: MUST FIX
+    install_requires=[
+        'minio <7.0.0, >=6.0.0',
+        'pytz',
+        'cached-property',
+        'attrdict'
+    ],
     classifiers=[
-        "Programming Language :: Python :: 3",
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8'
     ],
-    python_requires='>=3.6',
```

Comment:
why dont u support 3.6?
what is required?

Reply:
from __future__ import annotations